lxd/main_callhook.go

```diff
@@ -1,21 +1,29 @@
 package main
```

Comment:
so a general question for this file is why is it printing to stdout and why do the errors have `\n` added to them?

Reply:
Is this so they get logged in the liblxc log file?

Reply:
yes, stdout is used for outputing to the liblxc log file.

Reply:
ok cool

Reply:
fixed